recipes/gecos/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "1.2.0" %}
+{% set number = "0" %}
```

Comment:
Can just remove the templating for number when using directly below:
```suggestion
```

Most recipes only templatize name and version.